cli/integration_tests/run_summary/strict_env.t

```diff
@@ -0,0 +1,206 @@
+Setup
+  $ . ${TESTDIR}/../_helpers/setup.sh
+  $ . ${TESTDIR}/../_helpers/setup_monorepo.sh $(pwd) strict_env_vars
+
+Set the env vars
+  $ export GLOBAL_VAR_PT=higlobalpt
+  $ export GLOBAL_VAR_DEP=higlobaldep
+  $ export LOCAL_VAR_PT=hilocalpt
+  $ export LOCAL_VAR_DEP=hilocaldep
+  $ export OTHER_VAR=hiother
+  $ export SYSTEMROOT=hisysroot
+
+Run as `infer`
+  $ rm -rf .turbo/runs
+  $ ${TURBO} run build --summarize > /dev/null
+  $ cat .turbo/runs/*.json | jq -r '.envMode'
```

Comment:
I understand that it's unnecessary if you're `rm -rf`'ing right before., but all the other tests explicitly get the first one with `head -n1`.

Reply:
You mentioned that this didn't guarantee ordering in some other test so I took this strategy.

Why was the ksuid ordering not guaranteed?

Reply:
I don't remember the details to dig it up again, but I'm 99% sure that was an incorrect statement and I solved the issue I was seeing a different way. I think it's still safe to rely on ksuid ordering. (That said, it's not a requirement since `rm && cat` already does what we need).